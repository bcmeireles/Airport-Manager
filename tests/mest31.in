a LIS Portugal Lisboa
a POR Portugal Lisboa

v TT6 LIS POR 14-02-2022 14:00 2:00 200
v TT7 LIS POR 15-02-2022 14:00 2:00 120
v TT6 LIS POR 17-02-2022 14:00 2:00 150


r TT6 17-02-2022 123456789A 148
r TT6 17-02-2022 123456789B 1
r TT6 17-02-2022 123456789C 1
r TT6 14-02-2022 123456789E 180
r TT6 14-02-2022 123456789D 20


e TT6
e 123456789A
r TT7 15-02-2022 123456789A 20
r TT6 14-02-2022

v TT6 LIS POR 18-02-2022 14:00 12:00 200
v TT6 LIS POR 17-02-2022 14:00 12:00 150
v TT6 LIS POR 19-02-2022 14:00 12:00 150


r TT6 17-02-2022 123456789D 148
r TT6 17-02-2022 123456789C 1
r TT6 17-02-2022 123456789B 1
r TT6 18-02-2022 123456789A 200


r TT6 14-02-2022


v

q