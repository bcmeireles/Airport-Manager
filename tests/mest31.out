airport LIS
airport POR
not found
TT6: flight does not exist
123456789A: flight reservation already used
TT6: flight does not exist
TT7 LIS POR 15-02-2022 14:00
TT6 LIS POR 18-02-2022 14:00
TT6 LIS POR 17-02-2022 14:00
TT6 LIS POR 19-02-2022 14:00
